docs/developer-guide/contributors-quickstart.md

```diff
@@ -25,6 +25,9 @@ git clone https://github.com/argoproj/argo-cd.git
 
 <https://docs.docker.com/engine/install/>
 
+#### Note for Mac users: 
```

Comment:
Are these instructions specific for Mac only? I'd imagine that this would need to be done for Linux as well?

Reply:
@blakepettersson I believe it's Docker Desktop specific. So title should be changed to 

```
#### Note for Docker Desktop users:
```